### PR TITLE
Edit WAT article

### PR DESCRIPTION
**Pull request by romdotdog**

### CODE REVIEW

File: Understanding WebAssembly better by learning WebAssembly-Text.md [hunk 1]
```diff
@@ -329,20 +325,20 @@ reading this article. A few things to note:
...
+ - The offset variable points at the address in memory where the number that will
    result from the addition of the two previous numbers will be stored,
-   thats why it is initially 8.
- - We do a right-bit-shift of `2` to the offset when comparing that to the
+   thats why it is initially 8. Two `i32`s take 8 bytes in memory.
+ - ~~We do a right-bit-shift of `2` to the offset when comparing that to the
    requested length of the sequence, because we are comparing it as an index.
    It is the same as dividing it by `4` and getting a full number instead of
-   a decimal.
+   a decimal.~~ TODO: Replace with division
```
Comment (romdotdog) [inline Understanding WebAssembly better by learning WebAssembly-Text.md:339]: Don't do this. The code needs to be understandable rather than performant

**Comment by fabricio-p** [comment]
> We are going to call the stack that is split between the parent frame and the new
stack the child frame.

I think you missunderstood what I was trying to say. I was trying to say that the stack that is splitted is the parent frame and the new stack that results from the split is the child frame

**Comment by romdotdog** [comment]
whoops! I didn't misunderstand, but rather that was a typo

Something more like
> We are going to call the the stack that existed before the split the parent stack and the new stack the child stack.

**Comment by fabricio-p** [comment]
The parent frame/stack still exists

**Comment by romdotdog** [comment]
> 
> 
> The parent frame/stack still exists

Of course, my revision doesn't imply that the parent stack stops existing

**Comment by fabricio-p** [comment]
yeah, but you said ^ that 
> the stack that existed before the split

And that can be confusing

**Comment by romdotdog** [comment]
How about
> We are going to call the old stack the parent stack and the new stack the child stack.

**Comment by fabricio-p** [comment]
I edited the article after merging the pr and I think it's better to have it like how it currently is

**Comment by romdotdog** [comment]
IMO what was before the edit was unclear

**Pull request merged**